docs/getting-started/clients/browser/biometric.mdx

```diff
@@ -25,7 +25,7 @@ _Supported:_
 _Not Supported:_
 
 - Firefox Versions 86 and earlier
-- Side-loaded MacOS Desktop Apps
+- Side-loaded macOS Desktop Apps
```

Comment:
⛏️ "Desktop Apps" shouldn't be capitalized either, or "Versions" ... probably some more of these.

Reply:
Good call. I'll make notes to circle back to that, possibly in a broader "make more capitalization more consistent" change—I think there were a few other things I noticed along those lines when I was reading through recently